README.md

```diff
@@ -47,17 +36,18 @@ void callbackDispatcher() {
 }
 
 Workmanager.initialize(
-    callbackDispatcher, //the top level function.
-    isInDebugMode: true //If enabled it will post a notificiation whenever the job is running. Handy for debugging jobs
+    callbackDispatcher, // The top level function, aka Flutter entry point
```

Comment:
> aka Flutter entry point

Not really.

Reply:
I agree ; I believe it should be named in another fashion.
Calling it a `Dispatcher` suggests that it actually dispatches something, which (I think) it does not ; it's definitely a Flutter entry point, as defined by :
`flutterEngine.run(withEntrypoint: flutterCallbackInformation.callbackName, libraryURI: flutterCallbackInformation.callbackLibraryPath)`

Reply:
I took the name from the blogpost that engineered the Background entrypoint.
So while I don't fully agree that the callbackDispatcher is a good name. It is name it is referred to in Dart. 
Naming it different would make it harder to Google stuff about it.

Reply:
https://medium.com/flutter/executing-dart-in-the-background-with-flutter-plugins-and-geofencing-2b3e40a1a124

Search for `callbackDispatcher`

Reply:
Offline discussion. Name retains `callbackDispatcher` 
Some other renames will be done to make the API more clearer for starters